go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/bits-and-blooms/bitset v1.7.0 // indirect
 	github.com/consensys/bavard v0.1.13 // indirect
 	github.com/consensys/gnark-crypto v0.12.1 // indirect
-	github.com/cosmos/ledger-cosmos-go v0.13.2 // indirect
+	github.com/cosmos/ledger-cosmos-go v0.12.4 // indirect
```

Comment:
[for reviewers] this was downgraded to match the version in celestiaorg/cosmos-sdk which matches the latest v0.46.x release of cosmos/cosmos-sdk.

Ref: 
1. https://github.com/celestiaorg/cosmos-sdk/pull/357/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R21
2. https://github.com/cosmos/cosmos-sdk/blob/7799bba7bc889288607576aa11655b5fc31a2da9/go.mod#L21